src/app/pages/data-protection/snapshot/snapshot-form/snapshot-form.component.ts

```diff
@@ -223,4 +225,22 @@ export class SnapshotFormComponent implements FormConfiguration, OnDestroy {
     delete value['begin'];
     delete value['end'];
   }
+
+  dataHandler(entity: EntityFormComponent): void {
+    // Setup cron_schedule
+    const schedule = entity.wsResponse.schedule;
+    if (Number(entity.wsResponse.id) !== Number(this.pk)) console.error({ id: entity.wsResponse.id, pk: this.pk });
+    const formatted = schedule.minute + ' ' + schedule.hour + ' ' + schedule.dom + ' ' + schedule.month + ' ' + schedule.dow;
+    const cronField = entity.formGroup.controls['cron_schedule'];
+    cronField.setValue(formatted);
+    entity.fieldConfig[5].value = formatted;
```

Comment:
index usage looks not very reliable. can it be more specific?

Reply:
Yep. I did update that for scrub form but apparently missed it on the snapshot form.  Nice catch.